### PR TITLE
Grafana Dashboard: add Precompute Epoch Transition panels

### PR DESCRIPTION
**Pull request by twoeths**
**Motivation**

+ Add Precompute Epoch Transition panels

Screen Shot 2021-11-09 at 16 53 49

### CODE REVIEW

File: docker/grafana/provisioning/dashboards/lodestar.json [hunk 1]
```diff
@@ -12606,17 +12606,250 @@
...
+              "placement": "bottom"
+            },
+            "tooltip": {
+              "mode": "multi"
+            }
+          },
+          "targets": [
+            {
+              "exemplar": false,
+              "expr": "lodestar_precompute_next_epoch_transition_result_total",
```
Comment (dapplion) [inline docker/grafana/provisioning/dashboards/lodestar.json]: This should be a rate

**Review by dapplion — APPROVED** [review]

**Comment by Stig0812** [comment]
https://github.com/ChainSafe/lodestar/commit/40d47c1121669a42e66bb4aec0865045c6467e55

File: docker/grafana/provisioning/dashboards/lodestar.json (hunk 1 above)
Comment (Stig0812) [reply docker/grafana/provisioning/dashboards/lodestar.json]: https://github.com/ChainSafe/lodestar/pull/3421

Comment (Stig0812) [reply docker/grafana/provisioning/dashboards/lodestar.json]: https://github.com/ChainSafe/lodestar/pull/3421#pullrequestreview-801004593

**Pull request merged**